Drafting documentation for the GitHub Organizations and Teams (#134)
samvera/maintenance: TEAMS.md

## Samvera Labs
| Team | Description |
| ---- | ----------- |
| [admins](https://github.com/orgs/samvera-labs/teams/admins) | Users with administrative privileges for the Samvera Labs Organization |
| [contributors](https://github.com/orgs/samvera-labs/teams/contributors) | Users with the privileges to manage releases for GitHub projects, as well as publish releases to RubyGems |
| [maintenance](https://github.com/orgs/samvera-labs/teams/maintenance) | Current, active members of the Component Maintenance Interest Group |

## Samvera Deprecated
| Team | Description |
| ---- | ----------- |
| [admins](https://github.com/orgs/samvera-deprecated/teams/admins) | Users with administrative privileges for the Samvera Deprecated Organization |
| [contributors](https://github.com/orgs/samvera-deprecated/teams/contributors) | Users with the privileges to manage releases for GitHub projects, as well as publish releases to RubyGems |
| [maintenance](https://github.com/orgs/samvera-deprecated/teams/maintenance) | Current, active members of the Component Maintenance Interest Group |

## Administrators (Admins)

> GitHub Organizations distinguish between [owners](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization) and [moderators](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-moderators-in-your-organization). Members of the `admins` Team are possess both of these privileges.

Reviewer: ```suggestion
GitHub Organizations distinguish between [owners](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/maintaining-ownership-continuity-for-your-organization) and [moderators](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-moderators-in-your-organization). Members of the `admins` Team possess both of these privileges.
```

> ## Contributors

Reviewer: Can you add something to this section about how this team is sync'd to rubygems so that members have push privileges as stated in the chart above?

> By default, all other [members](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#organization-members) of an Organization should belong to the `contributors` Team. As there are currently no requirement to be added as a `member` to any of the Samvera Organizations, there is no existing process for admitting new `members`. Additionally, members of the `contributors` Team possess the privileges to push new releases of Samvera Gems to [RubyGems](https://rubygems.org/). At this point in time, should Samvera Community members require these privileges, it would be please be requested that this request be submitted to the [#dev Channel on the Samvera Slack](https://samvera.slack.com/app_redirect?channel=dev).

Reviewer: What do you think about this small change?
```suggestion
By default, all other [members](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#organization-members) of an Organization should belong to the `contributors` Team. As there are currently no requirement to be added as a `member` to any of the Samvera Organizations, there is no existing process for admitting new `members`. Additionally, members of the `contributors` Team possess the privileges to push new releases of Samvera Gems to [RubyGems](https://rubygems.org/). Samvera Community members requiring these privileges can request them on the [#dev Channel on the Samvera Slack](https://samvera.slack.com/app_redirect?channel=dev).
```
